index.html

```diff
@@ -142,15 +151,15 @@ <h2 class="heading">About
           <li><a href="#"><img src="images/social/github.svg" alt="github"></a></li>
           <li><a href="#"><img src="images/social/vector.svg" alt="vector"></a></li>
         </ul>
-        <button class="bg round-borders cap resume" type="button">Get my resume</button>
+        <button class="btn bg round-borders cap resume" type="button">Get my resume</button>
       </div>
-
+        
+      <div class="qualifications">

```

Comment:
- [ ] Currently, your qualification section does not match your chosen template. Consider adding the `angle arrow` and `border-bottom` where necessary so that your work matches that of your chosen template.

Your design:
<img width="359" alt="chrome_6HNupNTON3" src="https://user-images.githubusercontent.com/81468675/144666600-8709c4f5-4215-437d-8b68-cec262fb56b9.png">

Figma design:
<img width="245" alt="chrome_bQwbx9Swss" src="https://user-images.githubusercontent.com/81468675/144669554-62582cec-a91e-4dda-b6c3-13c456a9c0ed.png">